script/install.sh

```diff
@@ -67,7 +66,7 @@ fi
 if [ "$ALPAKA_CI_OS_NAME" = "Linux" ]
 then
     # Minimize docker image size
-    sudo apt-get --quiet --purge autoremove
+    sudo apt-get --quiet --purge --yes autoremove
     sudo apt-get clean
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 fi
```

Comment:
You can delete this whole `if-fi` block because it is not useful as we do not cache the image anymore.

Reply:
will do